tests/app/va/va_profile/test_va_profile_v3.py

```diff
@@ -0,0 +1,107 @@
+import json
+import pytest
+
+
+from app.va.va_profile import (
+    VAProfileClient,
+)
+from app.models import RecipientIdentifier
+from app.va.identifier import IdentifierType, transform_to_fhir_format, OIDS
+
+
+MOCK_VA_PROFILE_URL = 'http://mock.vaprofile.va.gov'
+
+
+@pytest.fixture(scope='function')
+def test_va_profile_client(mocker, notify_api):
```

Comment:
we have a testing style guide for test names [here](https://github.com/department-of-veterans-affairs/vanotify-team/blob/master/Process/testing_guide.md) - @k-macmillan I assume we want this new work to follow that convention?

Reply:
I agree, we should be following the testing guide with test names. This is a fixture though and should be probably be titled `mock_va_profile_client`

Reply:
good call. fixing.